test-utils/src/dev/test_cmds.rs

```diff
@@ -125,7 +125,51 @@ pub fn error_for_enoent() -> String {
 /// invocation to invocation (e.g., assigned TCP port numbers, timestamps)
 ///
 /// This allows use to use expectorate to verify the shape of the CLI output.
-pub fn redact_variable(input: &str) -> String {
+#[derive(Clone, Debug)]
+pub struct Redactor<'a> {
+    redact_uuids: bool,
+    extra: ExtraRedactions<'a>,
+}
+
+impl<'a> Redactor<'a> {
+    pub fn new() -> Self {
+        Self { redact_uuids: true, extra: ExtraRedactions::new() }
+    }
+
+    pub fn redact_uuids(&mut self, redact_uuids: bool) -> &mut Self {
+        self.redact_uuids = redact_uuids;
+        self
+    }
```

Comment:
At this point I'm bikeshedding but I feel like the methods are overly wordy [1/2]:

```suggestion
    pub fn uuids(&mut self, uuids: bool) -> &mut Self {
        self.uuids = uuids;
        self
    }
```

Reply:
maybe? I guess it matches `extra`

Reply:
Made this change.